src/site/content/en/payments/life-of-a-payment-transaction/index.md

```diff
@@ -224,6 +236,61 @@ request.addEventListener('paymentmethodchange', e => {
 });
 ```
 
+### Shipping address change event
+
+A payment app can optionally provide the customer's shipping address. This is
+convenient for customers because they don't have to manually enter any details
+into a form and they can store their shipping address in their preferred payment
+apps, rather than on multiple different merchant websites.
+
+If a customer updates their shipping address in a payment app after the
+transaction has been initiated, a shipping address change event will be emitted
+to the merchant. This helps the merchant determine the shipping cost based on
+the new address, update the total price, and return it back to the payment app.
+
+```js
+request.addEventListener('shippingaddresschange', e => {
+  e.updateWith({
+    // Update the details
+  });
+});
+```
+
+If the merchant can't ship to the updated address, they can provide an error
+message by adding an error parameter to the transaction details returned to the
+payment app.
+
+{% Aside %}
+Merchants don't receive customers' full shipping address until they've
+authorized the payment.
+{% endAside %}
+
+### Shipping option change event
+
+A merchant can offer multiple shipping options to the customer and can delegate
+that choice to the payment app. The shipping options are displayed as a list of
+prices and service names the customer can select from. For example:
+
+* Standard shipping - Free
+* Express shipping - $5
```

Comment:
For clarity's sake, I'd much rather we use the [ISO 4217 currency codes](https://en.wikipedia.org/wiki/ISO_4217) for examples and such.

A lot of places use dollar signs and there's also confusion between one stroke and two strokes.